<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPanel</class>
 <widget class="QDialog" name="ChatPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatPanel</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>User Name : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="chat">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>461</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="chatField">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourceIP">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sourcePort">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>140</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>8888</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destIP">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>230</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="destPort">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>8887</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>280</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectButton">
   <property name="geometry">
    <rect>
     <x>682</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
